examples/blog/layouts/partials/header.html

```diff
@@ -1,12 +1,13 @@
 <!doctype html>
 <html lang="en">
+
 <head>
     {{ partial "meta.html" . }}
 
     <title>{{ .Title }} - {{ .Site.BaseURL }}</title>
     <link rel="canonical" href="{{ .Permalink }}">
     {{ partial "header.includes.html" . }}
     {{ with .OutputFormats.Get "RSS" -}}
-    {{ printf "<link href=%q rel=\"alternate\" type=%q title=%q />" .Permalink .MediaType .Title | safeHTML }}
+    {{ printf "<link href=%q rel=\"alternate\" type=%q title=%q />" .Permalink .MediaType $.Site.Title | safeHTML }}
```

Comment:
I suspect the "correcter" fix in the above would be tu use `$.Title`.

Reply:
Okay, I will make the change. Do you want me to do a second rebase after committing or can I amend the commit and force push to my branch?

Reply:
Fix, squash, and force push.